finder-app/finder-test.sh

```diff
@@ -22,7 +22,7 @@ then
 else
 	NUMFILES=$1
 	WRITESTR=$2
-	WRITEDIR=/tmp/aeld-data/$3
+	WRITEDIR=~/tmp/aeld-data/$3
```

Comment:
@mash9756 was this change intentional?  I'm not sure it will work with assignment 3 and later.

Reply:
This change should have been reverted, I had added the ~ when debugging my code.